src/lib/components/Carousel/Carousel.tsx

```diff
@@ -43,17 +43,18 @@ export const Carousel: FC<CarouselProps> = ({
 
   const navigateTo = useCallback(
     (item: number) => () => {
+      if (!items) return;
       item = (item + items.length) % items.length;
       if (carouselContainer.current) {
         carouselContainer.current.scrollLeft = carouselContainer.current.clientWidth * item;
       }
       setActiveItem(item);
     },
-    [items.length],
+    [items],
   );
 
   useEffect(() => {
-    if (carouselContainer.current && !isDragging) {
+    if (carouselContainer.current && !isDragging && carouselContainer.current.scrollLeft !== 0) {
```

Comment:
Have to add additional condition to make tests work in the correct way, otherwise during the tests we have 
```typescript
activeItem = NaN
````
that ruins all calculation that depends on `activeItem`
the reason that during the tests
`carouselContainer.current.scrollLeft = 0` and `carouselContainer.current.clientWidth = 0` so `0 / 0 is NaN`